### PR TITLE
[presto] Avoid progress report exception when query just stopped

### PR DESCRIPTION
**Pull request by romainr**

### CODE REVIEW

File: desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py [hunk 1]
```diff
@@ -307,19 +307,24 @@ def check_status(self, notebook, snippet):
...
     return response
 
+
   @query_error_handler
   def progress(self, notebook, snippet, logs=''):
+    progress = 50
     if self.options['url'].startswith('presto://'):
       guid = snippet['result']['handle']['guid']
       handle = CONNECTIONS.get(guid)
-      if not handle:
```
Comment (penggongkui) [inline desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py:315]: Maybe this is also necessary, i am not sure.

Comment (romainr) [reply desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py:315]: Adding back!

Comment (penggongkui) [reply desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py:315]: When stats = None, now will throw exception? how about change back use return

Comment (romainr) [reply desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py:315]: If stats is None it should return 100 IIUC (line #324)

Comment (penggongkui) [reply desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py:315]: But 325 line, will throw none type has no gets function

Comment (romainr) [reply desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py:315]: Ha indeed, will fix soon!

Comment (romainr) [reply desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py:315]: https://github.com/cloudera/hue/pull/1560

**Pull request merged**